src/lib/tdtool.js

```diff
@@ -59,7 +62,7 @@ const TDtool = {
   on:  id => TDtool.run('--on', id),
   off: id => TDtool.run('--off', id),
   dim: id => TDtool.run('--dim', id),
-  dimlevel: level => TDtool.run('--dimlevel', level),
+  dimlevel: (level, id) => TDtool.run('--dimlevel', level, '--dim', id),
```

Comment:
A better option to me would be removing `dimlevel`, and implementing `dim` with the `execute`-implementation:
```js
dim: (level, id) => TDtool.isInstalled().then(() =>
      execute(`tdtool --dimlevel ${level} --dim ${id}`)),
```

This will keep the original `run` implementation and not break the existing actions.